addon-test-support/helpers.js

```diff
@@ -189,10 +189,10 @@ export function find(selector, contextEl) {
   } else if (Object.prototype.toString.call(selector) === "[object String]") {
     selected = document.querySelectorAll(`${settings.rootElement} ${selector}`);
   }
-  if (!settings.useJQueryWrapper) {
-    return (selected.length === 1) ? selected[0] : selected;
+  if (settings.useJQueryWrapper) {
```

Comment:
I definitely want to use this addon as a mechanism to use the same helpers from both acceptance and integration tests, however we should not try to serve both API's.  I want to use this addon as a way to start the gradual migration towards the future "testing unification" RFC setup.  That RFC proposes only native DOM.

I do not really want to try to make this a pure drop in for acceptance tests today, instead the idea is that this addon will support both acceptance and integration tests today with native DOM API's, and existing apps / addons can migrate towards it. The end state will be one step closer to landing "testing unification"...